dna/zomes/projects/src/lib.rs

```diff
@@ -63,6 +63,48 @@ pub enum SignalType {
   GoalVote(GoalVoteSignal),
   Member(MemberSignal),
   ProjectMeta(ProjectMetaSignal),
+  EditingGoal(EditingGoalSignal),
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub enum GoalField {
+  Title,
+  Description,
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub struct EditingGoalSignal {
+  pub goal_field: GoalField,
+  pub goal_address: HeaderHash,
+  pub editing_agent: AgentPubKey,
+  pub is_editing: bool,
+}
+
+#[derive(Debug, Serialize, Deserialize, SerializedBytes)]
+pub struct EditingGoalInfo {
```

Comment:
Let's use `EditingGoalInput` to follow a convention of calling a struct that's just used for data input from a client call an 'Input'